core/src/test/java/org/apache/iceberg/avro/TestAvroNameMapping.java

```diff
@@ -102,6 +102,7 @@ public void testMapProjections() throws IOException {
                             Types.NestedField.required(1, "lat", Types.FloatType.get()))))));
 
     projected = writeAndRead(writeSchema, readSchema, record, nameMapping);
+    @SuppressWarnings("unchecked")
```

Comment:
I think we can directly place @SuppressWarnings on top of the method, so we only need 1

Reply:
oh I see this comes from 2 methods. But even in that case I think it's always good practice to use annotation only in method declaration if possible

Reply:
Thank you for your suggestion. Since there are in total 5 unchecked warnings (2 Map types and 3 List types). I think it may be better to place one SuppressWarnings on top of the class to eliminate all the unchecked warning messages.

Reply:
👌